pkg/dispatchcli/cmd/install_config.go

```diff
@@ -52,7 +52,7 @@ dispatch:
   certSecret: dispatch-tls
   image:
     host: vmware
-    tag: dev-xxxx
+    tag: v0.1.2
```

Comment:
version bump to v0.1.2  should be done on [L46](https://github.com/vmware/dispatch/pull/110/files#diff-04c6e90faac2675aa89e2176d2eec7d8R46) and [L65](https://github.com/vmware/dispatch/pull/110/files#diff-04c6e90faac2675aa89e2176d2eec7d8R46) as well.

Reply:
thanks!